### ISSUE TITLE
Update loss function factory to be compatible with pytorch-metric-learning v0.9.90

### ISSUE THREAD

**Issue by KevinMusgrave** [KevinMusgrave/powerful-benchmarker#54]
Need to include getters for distance, reducer, weight regularizer, and embedding regularizer arguments.
